workflow-notifier/main.py

```diff
@@ -17,25 +17,25 @@ def get_env_or_fail(var_name: str) -> str:
     return value
 
 
-IDA_SERVER_URL = get_env_or_fail("IDA_SERVER_URL")
+SARA_SERVER_URL = get_env_or_fail("IDA_SERVER_URL")
```

Comment:
```suggestion
SARA_SERVER_URL = get_env_or_fail("SARA_SERVER_URL")
```

Not sure if i could do this or we would get problems now